compiler/test/compilable/test5973.d

```diff
@@ -25,7 +25,7 @@ class F : E
 {
     void test()
     {
-        i = 5;
```

Comment:
`fixAliasThis` is supposed to change the search order so ambiguities are resolved differently. It's not supposed to turn single `alias this` hits into errors, however weird the `sizeof.max` turning into `typeof(this.extent.sizeof).max` may be. The failure in dmd + buildkite projects suggests the implementation bugged and the switch is not ready to become the default yet.

Reply:
I thought we agreed that we weren't going to fix alias this, that it was unfixable, and we should just stick with the way it is now? (The reason is that fixing it just breaks something else.)

Reply:
FWIW this appears to only affect `alias this` hits within the `struct/(sub)class` that defines it and only when the hit is implicitly `[this.]foo` as opposed to `other.foo` as in [`auto getSource() out(result; result.isMixin() == isMixin())`](https://github.com/snazzy-d/sdc/blob/master/src/source/location.d#L168), the `result.isMixin()` is fine, the naked `isMixin()` is apparently not.

Also,
```d
new F().i;
```
would work perfectly fine here. Why this happens I do not know and defer to @RazvanN7 . If that were to be fixed (which on the surface seems to be an orthogonal issue) then this particular change would not need to be made.